client.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/lytics/grid/codec"
 	"github.com/lytics/grid/registry"
 	"github.com/lytics/retry"
-	etcdv3 "github.com/coreos/etcd/clientv3"
```

Comment:
Thanks, etcd :/ - They re-namespaced for a github organization change due to CoreOS changes, which is fine. What's not fine is that types are fully qualified, so they changed all their types at the same time. Upgrading just requires re-namespacing and everything is fine.